x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/indexlifecycle/LifecyclePolicyTests.java

```diff
@@ -41,7 +41,7 @@
 
     @Override
     protected LifecyclePolicy doParseInstance(XContentParser parser) {
-        return LifecyclePolicy.parse(parser, lifecycleName);
+        return LifecyclePolicy.parse(parser, lifecycleName, TestLifecycleType.INSTANCE);
```

Comment:
We should be able to rewrite the `createTestInstance()` and `mutateInstance()` methods so they use the timeseries lifecycle type so we don't have to have the parse method take the type here. The other tests in this suite can still use the TestLifecycleType since they create the policy directly rather than parsing from JSON

Reply:
OK